datetime/datetime.py

```diff
@@ -914,6 +914,11 @@ class tzinfo:
     Subclasses must override the name(), utcoffset() and dst() methods.
     """
     __slots__ = ()
+
+    def __new__(cls):
```

Comment:
If you run the test suite this will fail at some point because more arguments are passed. I'd suggest you go with 
```
    def __new__(cls, *args, **kwargs):
        return object.__new__(cls)
```

Reply:
thank you ,that works